playbooks/roles/ansible-role-django-ida/templates/defaults/main.yml.j2

```diff
@@ -29,6 +29,7 @@
     PORT: '3306'
     ATOMIC_REQUESTS: true
     CONN_MAX_AGE: 60
+    OPTIONS: '{{ '{{' }} {{ role_name|upper}}_MYSQL_OPTIONS }}'
```

Comment:
I guess from looking at this that the inner '{{' doesn't need to be escaped? I'm not clear on what purpose it serves